clusters/build-clusters/01_cluster/ci/_origin-release-build/golang-1.14/Dockerfile

```diff
@@ -30,8 +30,8 @@ ADD *.rpm /tmp/origin-rpm/
 RUN yum install -y /tmp/origin-rpm/*.rpm && \
     yum clean all && \
     curl https://storage.googleapis.com/golang/go$VERSION.linux-$(uname -m | sed 's/x86_64/amd64/g').tar.gz | tar -C /usr/local -xzf - && \
-    go get golang.org/x/tools/cmd/cover \
```

Comment:
@bear-redhat do you want to set `GO111MODULE=on` here for every tool?

Reply:
You can define this env var in https://github.com/openshift/release/pull/35404/files#diff-db17d8363092663a1244683ec26403b37ad044236fc87037e005394f238cb376L9 to avoid changing multiple lines

Reply:
Maybe something like `ENV GO111MODULE=on`

Reply:
`ENV GO111MODULE=on` leads to some errors that would solve by doing
```
GO111MODULE=off go get github.com/Masterminds/glide \
        github.com/openshift/imagebuilder/cmd/imagebuilder
```
Do we still want to do that?